### PR TITLE
West Virginia Subtractions

### LINKED ISSUE #3143: West Virginia Subtractions
West Virginia Subtractions

reference: https://tax.wv.gov/Documents/PIT/2022/PersonalIncomeTaxFormsAndInstructions.2022.pdf

### PR DESCRIPTION
**Pull request by Kiki-67**
Fixes #3143

### Summary
🆕🗂️📝

Add parameter file for West Virginia income tax subtractions. The file `policyengine_us/parameters/gov/states/wv/tax/income/subtractions.yaml` defines the income sources that are excluded from state taxable income.

> _`West Virginia` file_
> _Subtracts some income sources_
> _Autumn leaves fall low_

### Walkthrough
*  Add a new parameter file for West Virginia subtractions from federal adjusted gross income ([link](https://github.com/PolicyEngine/policyengine-us/pull/3144/files?diff=unified&w=0#diff-46c948599341c611749307704fa6095d4750740748cb0bf2edc469b63fad9b97R1-R19))

### CODE REVIEW

File: changelog_entry.yaml
Comment (PavelMakarchuk) [inline changelog_entry.yaml:1]: need to add changelog_entry

File: changelog_entry.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- bump: minor
+  changes:
+    added:
+    - west virginia subtractions.
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:4]:
```suggestion
    - West Virginia subtractions.
```

File: policyengine_us/parameters/gov/states/wv/tax/income/subtractions/subtractions.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/wv/tax/income/subtractions/subtractions.yaml:1]: create a wv_subtractions.py and make an adds function with this parameter file

File: policyengine_us/parameters/gov/states/wv/tax/income/subtractions/subtractions.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+description: West Verginia subtracts these sources from adjusted gross income.
+values:
+  2021-01-01:
+    - us_govt_interest # Line 27
+    - military_retirement_pay # Line 30
+    - wv_public_pension_subtraction.py # Line 31
+    - wv_social_security_benefits_subtraction # Line 32
+    - military_service_income # Line 34
+    - salt_refund_income # Line 36
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/wv/tax/income/subtractions/subtractions.yaml:9]: include: `investment_in_529_plan`

**Review by PavelMakarchuk — APPROVED** [review]

File: policyengine_us/parameters/gov/states/wv/tax/income/subtractions/subtractions.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+description: West Verginia subtracts these sources from adjusted gross income.
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/wv/tax/income/subtractions/subtractions.yaml:1]:
```suggestion
description: West Virginia subtracts these sources from adjusted gross income.
```

File: policyengine_us/parameters/gov/states/wv/tax/income/subtractions/subtractions.yaml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    - wv_public_pension_subtraction.py # Line 31
+    - wv_social_security_benefits_subtraction # Line 32
+    - military_service_income # Line 34
+    - salt_refund_income # Line 36
+    - investment_in_529_plan
+    #- SENIOR CITIZEN OR DISABILITY DEDUCTION. # Line 46
+
+metadata:
+  unit: list
+  label: West Virginia subtractions from federal adjusted gross income
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/wv/tax/income/subtractions/subtractions.yaml:15]:
```suggestion
  label: West Virginia subtractions
```

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**